### ISSUE TITLE
Change to use Stubble (fork of Nustache)

### ISSUE THREAD

**Issue by dennisroche** [baldator/Poshstache#1]
The original authors of Nustache have forked a new library that is compatible with `netstandard1.3`.

https://github.com/StubbleOrg/Stubble

Also, the CI build for this project looks like it is silently failing.

**Comment by baldator** [comment]
Hi @dennisroche 
Thank you for highlighting these points. I've modified the code to use stubble and fixed the CI script.

Marco

**Comment by geographika** [comment]
@baldator - I'm not sure the Stubble DLL can be dropped in without further changes. 

I get `Unable to find type [Stubble.Core.Render]` when using ConvertTo-PoshstacheTemplate v0.1.3

It fails in ConvertTo-PoshstacheTemplate.ps1 on the following line:
`
return [Stubble.Core.Render]::StringToString($InputString, $JsonInput)
`

Nustache has a StringToString function but it looks like the Stubble API is different, see [test suite here](https://github.com/StubbleOrg/Stubble/blob/master/test/Stubble.Core.Tests/Renderers/StringRenderer/RendererTests.cs)

Switching back to the following works fine:

```
    [Reflection.Assembly]::LoadFile("$Path\binary\Nustache.Core.dll") | Out-Null
    try{
        return [Nustache.Core.Render]::StringToString($InputString, $JsonInput)
    } catch [Exception] {
        $_.Exception.Message
    }
```

**Comment by baldator** [comment]
Thank you for the feedback @geographika, much appreciated. I've rollbacked to Nustache and published version v0.1.4. I'm going to create a new repository for Stubble.

**Comment by whihathac** [comment]
Is this issue fixed? I can't use Stubble in PS ver 5.x?
